webrender_traits/src/api.rs

```diff
@@ -207,8 +207,8 @@ impl RenderApi {
     }
 
     /// Translates a point from viewport coordinates to layer space
-    pub fn translate_point_to_layer_space(&self, point: &Point2D<f32>)
-                                          -> (Point2D<f32>, PipelineId) {
+    pub fn translate_point_to_layer_space(&self, point: &LayoutPoint)
```

Comment:
Good catch, I think that it should take a WorldPoint and return a LayoutPoint. I made the change locally, that said the implementation is ```panic!("unused api - remove from webrender_traits");``` so I guess it won't stick around for long.